Jenkinsfile

```diff
@@ -298,6 +298,38 @@ pipeline {
             }
           }
         }
+        /**
+        Publish a snapshot. A "snapshot" is a packaging of the latest *unreleased* APM agent,
+        published to a known GCS bucket for use in edge demo/test environments.
+        */
+        stage('Publish snapshot') {
+          options { skipDefaultCheckout() }
+          environment {
+            BUCKET_NAME = 'oblt-artifacts'
+            BUILD_NODE_VERSION = '14'
+            DOCKER_REGISTRY = 'docker.elastic.co'
+            DOCKER_REGISTRY_SECRET = 'secret/observability-team/ci/docker-registry/prod'
+            GCS_ACCOUNT_SECRET = 'secret/observability-team/ci/snapshoty'
+          }
+          // when { branch 'main' }
```

Comment:
Did you forget to uncomment this? Or should it be removed entirely?

Reply:
Good catch. It wasn't part of the last re-commit.